lib/src/widgets/chat/ds_quick_reply.widget.dart

```diff
@@ -0,0 +1,122 @@
+import 'dart:convert';
+
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSQuickReply extends StatelessWidget {
+  final DSAlign align;
+  final Map<String, dynamic> content;
+  final Function? onSelected;
+  final bool hideOptions;
+
+  const DSQuickReply({
+    Key? key,
+    required this.align,
+    required this.content,
+    this.onSelected,
+    this.hideOptions = false,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: align == DSAlign.left
+          ? CrossAxisAlignment.start
+          : CrossAxisAlignment.end,
+      children: [
+        DSTextMessageBubble(
+          text: content['text'],
+          align: align,
+        ),
+        Align(
+          alignment: Alignment.bottomLeft,
+          child: _buildQuickReply(),
+        )
+      ],
+    );
+  }
+
+  Widget _buildQuickReply() {
+    return hideOptions ? const SizedBox() : _buildItems();
+  }
+
+  Widget _buildItems() {
+    List<DSBorderRadius> borderRadius = [];
+    List<Widget> children = [];
+
+    borderRadius = [];
+
+    content['options'].forEach(
+      (dynamic option) {
+        children.add(
+          GestureDetector(
+            onTap: () {
+              if (onSelected != null) {
+                Map<String, dynamic> payload = {};
+
+                if (option.containsKey('value')) {
+                  String type = option['type'];
+                  payload = {
+                    "type": type,
+                    "content": type.contains('json')
+                        ? jsonDecode(option['value'])
+                        : option['value']
+                  };
+                } else {
+                  payload = {
+                    "type": 'text/plain',
+                    "content": option.containsKey('order')
+                        ? option['order'].toString()
+                        : option['text']
```

Comment:
I suggest using a comma at this point. depending on the editor configuration, the indentation may break.